src/application/F3DOptionsParser.cxx

```diff
@@ -179,101 +174,101 @@ class ConfigurationOptions
+    this->DeclareOption(grp2, "line-width", "", "Line width when showing edges", options.getAsDoubleRef("line-width"), true, true, "<width>");
+    this->DeclareOption(grp2, "color", "", "Solid color", options.getAsDoubleVectorRef("color"), true, true, "<R,G,B>");
+    this->DeclareOption(grp2, "opacity", "", "Opacity", options.getAsDoubleRef("opacity"), true, true, "<opacity>");
+    this->DeclareOption(grp2, "roughness", "", "Roughness coefficient (0.0-1.0)", options.getAsDoubleRef("roughness"), true, true, "<roughness>");
+    this->DeclareOption(grp2, "metallic", "", "Metallic coefficient (0.0-1.0)", options.getAsDoubleRef("metallic"), true, true, "<metallic>");
+    this->DeclareOption(grp2, "hdri", "", "Path to an image file that will be used as a light source", options.getAsStringRef("hdri"), false, true, "<file path>");
+    this->DeclareOption(grp2, "texture-base-color", "", "Path to a texture file that sets the color of the object", options.getAsStringRef("texture-base-color"), false, true, "<file path>");
+    this->DeclareOption(grp2, "texture-material", "", "Path to a texture file that sets the Occlusion, Roughness and Metallic values of the object", options.getAsStringRef("texture-material"), false, true, "<file path>");
+    this->DeclareOption(grp2, "texture-emissive", "", "Path to a texture file that sets the emitted light of the object", options.getAsStringRef("texture-emissive"), false, true, "<file path>");
+    this->DeclareOption(grp2, "emissive-factor", "", "Emissive factor. This value is multiplied with the emissive color when an emissive texture is present", options.getAsDoubleVectorRef("emissive-factor"), true, true, "<R,G,B>");
+    this->DeclareOption(grp2, "texture-normal", "", "Path to a texture file that sets the normal map of the object", options.getAsStringRef("texture-normal"), false, true, "<file path>");
+    this->DeclareOption(grp2, "normal-scale", "", "Normal scale affects the strength of the normal deviation from the normal texture", options.getAsDoubleRef("normal-scale"), true, true, "<normalScale>");
 
     auto grp3 = cxxOptions.add_options("Window");
-    this->DeclareOption(grp3, "bg-color", "", "Background color", options.BackgroundColor, true, true, "<R,G,B>");
-    this->DeclareOption(grp3, "resolution", "", "Window resolution", options.Resolution, true, true, "<width,height>");
-    this->DeclareOption(grp3, "fps", "z", "Display frame per second", options.FPS);
-    this->DeclareOption(grp3, "filename", "n", "Display filename", options.Filename);
-    this->DeclareOption(grp3, "metadata", "m", "Display file metadata", options.MetaData);
-    this->DeclareOption(grp3, "fullscreen", "f", "Full screen", options.FullScreen);
-    this->DeclareOption(grp3, "blur-background", "u", "Blur background", options.BlurBackground);
+    this->DeclareOption(grp3, "bg-color", "", "Background color", options.getAsDoubleVectorRef("background-color"), true, true, "<R,G,B>"); // TODO change name ?
```

Comment:
`TODO`: yes, please use `bg-color` in `f3d::options`

Reply:
tbh i dont like bg.

Reply:
Yes, but the command line option is `bg-color`, and for backward compatibility it would be better to not rename it.

Reply:
not in this MR though.